artifacts/setup_alfresco.py

```diff
@@ -0,0 +1,283 @@
+import base64
+import requests
+import json
+
+# Template CSV file looks like
+# line 1: Base URL, username, password
+# line 2: user1, user2, ...
+# line 3: group1, group2, ...
+# line 4: public site1, public site2, ...
+# line 5: rm site
+# line 6: folder1, folder2
+# line 7: rm root category
+# line 8: rm category1, rm category2
+
+# For example:
+# http://localhost:8080,admin,admin
+# yasser
+# ARKCASE_ENTITY_ADMINISTRATOR,ARKCASE_CONSUMER,ARKCASE_SUPERVISOR,ARKCASE_ADMINISTRATOR,ARKCASE_EXTERNAL,ARKCASE_CONTRIBUTOR
+# acm
+# rm
+# Case Files,Complaints,Document Repositories,Expenses,People,Recycle Bin,Tasks,Timesheets,User Profile,Business Processes,Consultations,SAR,Requests
+# ACM
+# Case Files,Complaints,Document Repositories,Requests,Tasks,Consultations,SAR
+
+content_services_path = '/alfresco/api/-default-/public/alfresco/versions/1'
+ags_services_path = '/alfresco/api/-default-/public/gs/versions/1'
+filename = './alfresco_setup.properties'
+
+
+def remove_eol(string: str):
```

Comment:
You can use `str.strip("\n")` instead of re-writing it yourself

Reply:
Using environment variables so the line feeds are no longer there